doc/includes/include_kconfig_common.txt

```diff
@@ -21,6 +21,10 @@
 
 * ``CONFIG_SIDEWALK_ON_DEV_CERT`` -- Enables the on-device certification Shell.
 
+* ``SIDEWALK_CRYPTO_PSA_KEY_STORAGE`` - Enables secure storage for persistent Sidewalk keys.
+
+* ``SIDEWALK_MFG_STORAGE_SUPPORT_HEX_v7`` - Enables support for old Sidewalk manufacturing hex format.

```

Comment:
Maybe it would be better to indicate the release version instead? "old" sounds very vague